### PR TITLE
Onboarding improvements - development workflow

### PR DESCRIPTION
**Pull request by AronNovak**
#486

### CODE REVIEW

**Comment by AronNovak** [comment]
^^ GitPod app integration is back

**Comment by AronNovak** [comment]
image

**Comment by AronNovak** [comment]
image

**Comment by AronNovak** [comment]
It's not yet perfect, contacted GitPod support on port visibility issues.

**Comment by AronNovak** [comment]
@anvmn It's half-ways.. GitPod is usable, however not 100%, and could not yet gain proper access to a Mac to work on the compatibility, working via CircleCI is highly ineffective for such a work, so I wait for ScaleWay to buy some more mac minis that I can use in the cloud :)

However finally we have much better PHP 7.4 compatibility as the tests are executed inside DDEV and uses PHP 7.4, not some older version.

File: README.md [hunk 1]
```diff
@@ -24,52 +24,86 @@ Primary ports:
 
 ### GitPod first steps
 1. Wait until the Drupal login page shows up
-2. Login with `admin` / `admin` into the Drupal backend.
-3. Choose "Remote Explorer" on the left and open port 3000 too, either in new browser window or in preview.
-4. Use `12345678` as the pairing code (tied to Device nodes at the Drupal side).
-5. Use `1234` as the PIN code (tied to the Nurse nodes at the Drupal side).
-6. Initiate a sync process. The device status page allows you to initiate a manual sync with the backend.
+1. Login with `admin` / `admin` into the Drupal backend.
+1. Choose "Remote Explorer" on the left and open port 3000 too, either in new browser window or in preview.
```
Comment (anvmn) [inline README.md:28]: Here and bellow, indexes do not increase for some reason.
We got:
```
1.
1.
1.
...
```

**Review by anvmn — CHANGES_REQUESTED** [review]
@AronNovak 

Something strange. PR says that README.md requires re-roll, but:

Selection_1175

File: README.md (hunk 1 above)
Comment (AronNovak) [reply README.md:28]: See https://www.markdownguide.org/basic-syntax/#ordered-lists - it's a technique to avoid re-writing numbers if you inject items.

**Review by anvmn — APPROVED** [review]
@AronNovak 

Shall we merge it?

**Pull request merged**